content/blog/microsoft-platinum-sponsor.md

```diff
@@ -0,0 +1,15 @@
+Title: Announcing Microsoft as Platinum Sponsor
+Date: 2017-10-29 03:45
+Category: 2017
+Slug: platinum-sponsor-Microsoft
+Summary: Microsoft as Platinum Sponsor
+
+[Microsoft](https://www.microsoft.com/en-in)'s commitment to openness and collaboration is ingrained in our day-to-day approach to doing business. We continue to invest in delivering a great experience for any developer, with any app, on any platform. From making the full server-side .NET stack open source to delivering Bash on Ubuntu on Windows and Visual Studio Code,a free full featured code editor for Linux, Mac, OSX and Windows, we reinforce the commitment.
+
+Our Cloud platform, [Microsoft Azure](https://azure.microsoft.com/en-in/) supports the broadest selection of operating systems, programming languages, frameworks, tools, databases and devices. Run Linux containers with Docker integration, build apps with JavaScript, Python, .NET, PHP, Java and Node.js, build back-ends for iOS, Android and Windows devices. Azure cloud service supports the same technologies millions of developers and IT professionals already rely on and trust.
+
+<img src="https://blogs.microsoft.com/wp-content/uploads/2012/08/8867.Microsoft_5F00_Logo_2D00_for_2D00_screen.jpg">
```

Comment:
please replace this address with [this](https://in.pycon.org/2017/images/sponsor/microsoft.png)